packages/rocketchat-livechat/server/publications/livechatRooms.js

```diff
@@ -0,0 +1,11 @@
+Meteor.publish('livechat:rooms', function(offset = 0, limit = 20) {
+	if (!this.userId) {
+		throw new Meteor.Error('not-authorized');
```

Comment:
You should probably start using throw new Meteor.Error('error-not-authorized', 'Not authorized', { method: 'livechat:rooms' });